src/node/file-tree-utils.js

```diff
@@ -279,6 +275,27 @@ export async function repopulateFileTree() {
 
   const outputFiles = files.filter((file) => !inputFiles.includes(file));
 
-  fileTreeEl.inputFiles = inputFiles;
   fileTreeEl.outputFiles = outputFiles;
+  fileTreeEl.inputFiles = inputFiles;
+
+  mightDispatchTokens();
+}
+
+let oldTokens = [];
+function mightDispatchTokens() {
+  // Naive compare of the tokens
+  if (
+    JSON.stringify(oldTokens) !== JSON.stringify(styleDictionaryInstance.tokens)
```

Comment:
why tokens and not properties ?

Reply:
not sure I used, the properties because it's the merged version of all properties, but I may be mistaken

Reply:
I thought that's what the tokens object was 😆

Reply:
I asked Danny Banks:
```
there is not, originally it was called properties, but then the community shifted to using the term design tokens so in 3.0 we added 'tokens', but wanted to be backwards compatible so they both mean the same thing
```